themes/hugo-rladies/layouts/events/single.html

```diff
@@ -1,335 +1,164 @@
+    var loadingEl = document.getElementById("loading");
+    var calendarEl = document.getElementById("calendar");
+    evnt = {
+      url: "https://raw.githubusercontent.com/rladies/website/main/data/events.json", //your url,
+      type: "GET",
+      dataType: "json",
+      success: function (data) {
+        //based on the dropdown changetimezone of each event
+        //console.log(data);
+
+        let updatedTime = [];
+        $.each(data, function (k, v) {
+          v.start = moment.tz(v.start, timeZone).format();
+          v.end = moment.tz(v.end, timeZone).format();
+          updatedTime[k] = v;
+        });
+        //console.log(Intl.DateTimeFormat().resolvedOptions().timeZone);
+        return updatedTime;
+      },
+    };
+    var calendar = new FullCalendar.Calendar(calendarEl, {
+      //schedulerLicenseKey: 'CC-Attribution-NonCommercial-NoDerivatives',
+      views: {
+        timeGridUseRWeek: {
+          type: "timeGridWeek",
+          //duration: { days: 5 },
+          visibleRange: {
+            start: "2019-12-17",
+            end: "2021-09-30",
+          },
+          buttonText: "week",
         },
-        "isReadOnly": true,
-        "usageStatistics": false,
-        "useNav": true,
-        "theme": rladies_cal,
-        "calendars":  [
-          {{ range .Site.Data.chapters }}
-            {
-              id: {{ .id }},
-              name: {{ .name | default "rladies" }},
-              color: {{ .color | default "#fff" }},
-              bgColor: {{ .bgColor | default "#88398a" }},
-              borderColor: {{ .color | default "#88398a" }},
-            },
-          {{ end }}
-        ] 
-      });
-  
-      function cal_change_ww(size){
-        var viewName = "";
-        var options = cal.getOptions();
-        switch (true) {
-            case (size < 650):
-                viewName = 'day';
-                break;
-            case (size < 900):
-                viewName = 'week';
-                break;
-            case (size < 1200):
-                options.month.visibleWeeksCount = 2;
-                viewName = 'month';
-                break;
+      },
+      headerToolbar: {
+        left: "prev,next",
+        center: "title",
+        right: "dayGridMonth,timeGridUseRWeek,listWeek,timeGridDay",
+      },
+      initialDate: "2021-01-04",
+      titleFormat: {
+        // will produce something like "Tuesday, September 18, 2018"
+        month: "long",
+        year: "numeric",
+        day: "numeric",
+      },
+      initialView: "dayGridMonth",
+      height: "auto",
+      //slotDuration: "00:30:00",
+      //defaultTimedEventDuration: "00:30:00",
+      navLinks: true, // can click day/week names to navigate views
+      editable: true,
+      selectable: true,
+      dayMaxEvents: true, // allow "more" link when too many events
+
+      eventDidMount: function (info) {
+        var tooltip = new Tooltip(info.el, {
+          title: info.event.extendedProps.description,
```

Comment:
```suggestion
          title: info.event.extendedProps.body,
```

I'll work on making that look nicer.